src/settings/JobProfiles/JobProfileDetails/JobProfileDetails.js

```diff
@@ -0,0 +1,136 @@
+                  <Row>
+                    <Col xs={9}>
+                      <Headline
+                        data-test-headline
+                        size="x-large"
+                        tag="h2"
+                        margin="small"
+                      >
+                        {jobProfile.name}
+                      </Headline>
+                    </Col>
+                    <Col xs={3}>
+                      <Row end="xs">
+                        <Col xs>
+                          <ExpandAllButton />
+                        </Col>
+                      </Row>
+                    </Col>
+                  </Row>
+                  <AccordionSet id="job-profile-details-accordions">
+                    <Accordion label={<FormattedMessage id="ui-data-export.summary" />}>
+                      <ViewMetaData
+                        metadata={jobProfile.metadata}
+                        stripes={stripes}
+                      />
+                      <Row>
+                        <Col xs>
+                          <KeyValue
+                            data-test-job-profile-name
+                            label={<FormattedMessage id="ui-data-export.name" />}
+                            value={jobProfile.name}
+                          />
+                        </Col>
+                      </Row>
+                      <Row>
+                        <Col xs>
+                          <KeyValue
+                            data-test-job-profile-mapping-profile
+                            label={<FormattedMessage id="ui-data-export.mappingProfile" />}
+                            value={mappingProfile.name}
+                          />
+                        </Col>
+                      </Row>
+                      <Row>
+                        <Col xs>
+                          <KeyValue
+                            data-test-job-profile-protocol
+                            label={<FormattedMessage id="ui-data-export.protocol" />}
+                            value={<NoValue />}
+                          />
+                        </Col>
+                      </Row>
+                      <Row>
+                        <Col xs>
+                          <KeyValue
+                            data-test-job-profile-description
+                            label={<FormattedMessage id="ui-data-export.description" />}
+                            value={jobProfile.description || <NoValue />}
+                          />
+                        </Col>
+                      </Row>
+                    </Accordion>
+                  </AccordionSet>
+                </AccordionStatus>
+              )
+            }
+          </FullScreenView>
+        </Layer>
+      )}
+    </FormattedMessage>
+  );
+};
+
+JobProfileDetails.propTypes = {
+  onCancel: PropTypes.func,
+  mappingProfile: PropTypes.object,
+  jobProfile: PropTypes.object,
+  stripes: PropTypes.object,
+  loading: PropTypes.bool,
```

Comment:
I think we should try to be consistent in our code. We use hasLoaded, isLoading in some places, but in other - loading. So, what is the right choice according to our code conventions?

Reply:
Let's go with `isLoading` then.